cmd/kubeadm/app/constants/constants.go

```diff
@@ -95,6 +95,10 @@ const (
 	// It's copied over to kubeadm until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
 	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
 
+	// LabelNodeRoleExcludeBalancer specifies that the node should be
+	// exclude from load balancers created by a cloud provider.
+	LabelNodeRoleExcludeBalancer = "node.role.kubernetes.io/exclude-balancer"
```

Comment:
Is there a reason why this label is `node.role` and the one above is `node-role`?

Reply:
@brendandburns